packages/components/src/components/radio-button/radio-button.css

```diff
@@ -14,13 +14,9 @@ scale-radio-button {
   --background-disabled: var(--telekom-color-ui-disabled);
   --color-error: var(--telekom-color-text-and-icon-functional-danger);
   --transition: all var(--telekom-motion-duration-transition)
-    /* FIXME:TOKEN update this */
     cubic-bezier(var(--telekom-motion-easing-standard));
```

Comment:
I prefer to leave `cubic-bezier` so it still works (until we fix the tokens package), but I removed the comment because it's not strictly related to dark mode :)